package com.kh.example.chap03_test.run;

import com.kh.example.chap03_test.controller.TestController;
import com.kh.example.chap03_test.model.TestException;

public class Run {

	public static void main(String[] args) throws TestException {
		new TestController().secondException();

	}

}
